pkg/clients/edgeconnect/client.go

```diff
@@ -215,13 +214,12 @@ func (c *client) GetEdgeConnect(edgeConnectId string) (GetResponse, error) {
 }
 
 // UpdateEdgeConnect updates existing edge connect hostPatterns and oauthClientId
-func (c *client) UpdateEdgeConnect(edgeConnectId, name string, hostPatterns []string, hostMappings []edgeconnectv1alpha1.HostMapping, oauthClientId string) error {
+func (c *client) UpdateEdgeConnect(edgeConnectId string, edgeConnectConfig *Request) error {
```

Comment:
If the var type is `*Request` I would just name it `request` instead of `edgeConnectConfig`

Reply:
Or rename the struct to `EdgeConnectConfig` maybe?